csm/conf/setup.yaml

```diff
@@ -46,7 +46,7 @@ csm:
     ha:
         script: /opt/seagate/cortx/ha/conf/script/build-ha-csm
         args:
-            - /opt/seagate/cortx/ha/conf/build-ees-ha-csm-args.yaml
+            - /opt/seagate/cortx/ha/conf/build-ha-csm-args.yaml
```

Comment:
Is this file part of HA or provisioner?
Has file name renamed by other component?

Reply:
Yes other components have made changes, intergration testing is pending. It will be done by making custom build.